sys/net/routing/aodvv2/aodv.c

```diff
@@ -57,15 +61,24 @@ static mutex_t rreq_mutex;
 static mutex_t rrep_mutex;
 static mutex_t rerr_mutex;
 
-struct netaddr na_mcast;
+struct netaddr na_mcast = (struct netaddr){};
```

Comment:
had to change this line to 
`struct netaddr na_mcast = {{0},0,0};`
for `native` and `iot-lab_M3` since my compilers [1] and [2] didn't like this kind of initialization:
```
RIOT/sys/net/routing/aodvv2/aodv.c:64:35: error: initializer element is not constant
 struct netaddr na_mcast = (struct netaddr){};
                                   ^
```

[1] gcc version 4.8.2 (Ubuntu 4.8.2-19ubuntu1)
[2] gcc version 4.9.3 20141119 (release) \[ARM/embedded-4_9-branch revision 218278\] (GNU Tools for ARM Embedded Processors)